Added Visualisation and Dashboard and updated date format.

amitbhoraniya opened the pull request:
- Added objects.ndjson file for dashboard and visulization
- Updated date format for testcase and suite start time. Now elastic
search consider those field as a Date field and can use different filter
in kibana

cjayswal reviewed (commented):
Need to remove `ElasticSerachService` class from pull request as nochanges there.

cjayswal commented on src/com/qmetry/qaf/automation/elasticsearch/ElasticSerachService.java:
```diff
@@ -49,12 +49,12 @@
  *
  */
 public class ElasticSerachService {
-	private static final ElasticSerachService SERVICE = new ElasticSerachService();
```
Looks like committed by mistake.

amitbhoraniya replied:
Actually intensely I have changed line order, because when it is going to create a new instance of ElasticSerachService, it uses INDEX_NAME variable which have been not initialised yet. So first it was creating index with 'null' name.

I have debug it and found that root cause only, still needs your input.

cjayswal commented on src/com/qmetry/qaf/automation/elasticsearch/TestCaseRunResultDocument.java:
```diff
@@ -43,26 +45,27 @@
 		name = result.getName();
 		status = result.getStatus().name();
-		stTime = result.getStarttime();
+		stTime = DATE_FORMAT.format(new Date(result.getStarttime()));
```
Use DateUtils instead.

amitbhoraniya replied:
updated with DateUtils

Commit:
- Added objects.ndjson file for dashboard and visulization
- Updated date format for testcase and suite start time. Now elastic
search consider those field as a Date field and can use different filter
in kibana
